### DISCUSSION TITLE
Clarification on --getpkgbuild

### DISCUSSION THREAD

**Discussion by DistractionRectangle** [Morganamilo/paru#1303] [Q&A]
The man says it `Downloads PKGBUILDs from the ABS or AUR. The ABS can only be used for Arch Linux repositories.` but as far as I can tell it only queries pulls from the AUR:

```
❯ paru --getpkgbuild grub
:: Querying AUR...
:: packages not in the AUR: grub
```

Has the behavior changed? Or is there some config/flag I'm supposed to flip to enable it?

**Comment by Morganamilo** [comment]
Works for me. Open a proper bug report so I can try reproduce according to your config.
